packages/espressocash_app/lib/features/ramp_partner/models/ramp_partner.dart

```diff
@@ -67,27 +79,36 @@ enum RampPartner {
     title: 'Brij Network (DEMO)',
     minimumAmount: r'$5',
     paymentMethods: [PaymentMethod.bank],
-    partnerPK: '9YmsP8PoWfNaTwBjLRy8R5Yr9Ukcu2hvVvzea8mRpnKp',
+    brijParams: BrijParams(
+      partnerPK: '9YmsP8PoWfNaTwBjLRy8R5Yr9Ukcu2hvVvzea8mRpnKp',
+      termsUrl: '',
+      privacyUrl: '',
+    ),
   ),
 
   scalexBrij(
     title: 'Scalex (via BRIJ)',
     minimumAmount: r'$5',
     paymentMethods: [PaymentMethod.bank],
-    partnerPK: '342yN5YEX6bgGaTCuprVquFXhpZ8twyPzBo5kMZWQpDQ',
+    brijParams: BrijParams(
+      partnerPK: '342yN5YEX6bgGaTCuprVquFXhpZ8twyPzBo5kMZWQpDQ',
+      termsUrl: 'https://www.scalex.africa/termsofuse',
```

Comment:
We should rather get this information from the server, let's extend partner data with these URLs.

Reply:
added prs for adding new fields:

https://github.com/espresso-cash/brij-storage-gw/pull/7
https://github.com/espresso-cash/brij-storage-grpc/pull/26
https://github.com/espresso-cash/brij-client-sdk-dart/pull/22